### ISSUE TITLE
Do not remove the new line after the assignment operator

### ISSUE THREAD

**Issue by rousskov** [Mark-Simulacrum/attractifier#8]
This formatting change is undesirable:

```
 response.finalize();
-const zeroNineBody =
-    "<html><h1>garbage:1</h1>\n" +
+const zeroNineBody = "<html><h1>garbage:1</h1>\n" +
     "<h2>garbage:2</h2></html>\n" +
     responsePrefix(response) +
     response.body.whole();
 ```
The original new line should be preserved. I cannot think of any reasonably general case where removing a new line after "=" would be usually better than allowing the developer to make that formatting decision.

**Comment by Mark-Simulacrum** [comment]
Please confirm that this is fixed as of version 1.6.3.

**Comment by rousskov** [comment]
Confirming as fixed.
